title.html

```diff
@@ -6,4 +6,4 @@
 <meta itemprop="name" content="Jitsi Meet"/>
 <meta itemprop="description" content="Join a WebRTC video conference powered by the Jitsi Videobridge"/>
 <meta itemprop="image" content="images/jitsilogo.png?v=1"/>
-<link rel="icon" type="image/png" href="images/favicon.ico?v=1"/>
+<link rel="icon" href="images/favicon.svg">
```

Comment:
Can you pl add the `?v=1` ? It will help with caching.

Reply:
Ok, I added it.